src/status_im/chat/sign_up.cljs

```diff
@@ -105,25 +105,35 @@
                 :to           "me"}])))
 
 ;; -- Saving password ----------------------------------------
-(defn passpharse-messages [mnemonic]
+(defn account-generation-message []
   (dispatch [:received-message
              {:message-id   (random/id)
-              :content      (label :t/here-is-your-passphrase)
+              :content      (label :t/account-generation-message)
               :content-type text-content-type
               :outgoing     false
               :chat-id      console-chat-id
               :from         console-chat-id
-              :to           "me"
-              :new?         false}])
+              :to           "me"}]))
+
+(defn passpharse-messages [mnemonic messages-count]
+  (dispatch [:received-message
+             {:message-id   (random/id)
+              :content      (if (> messages-count 3)
+                              (label :t/phew-here-is-your-passphrase)
+                              (label :t/here-is-your-passphrase))
```

Comment:
But there can be a situation when you have more than three messages, but need to display `:t/here-is-your-passphrase` message without "Phew" at the beginning.
Not sure that it is important, but probably it is not the best way to determine which phrase to show.